service/complete_swap_transaction.js

```diff
@@ -65,13 +65,13 @@ module.exports = (args, cbk) => {
     },
 
     // Initialize the LN daemon connection
-    lnd: cbk => {
+    lnd: ['validate', ({}, cbk) =>  {
       try {
-        return cbk(null, lightningDaemon({}));
+        return cbk(null, lightningDaemon({'network':args.network}));
```

Comment:
I made a mistake in the earlier review

This network argument will return the wrong network, the network in args is the chain network, but you want the invoice network. That can be derived from the parseInvoice result so it should be invoice.network not args network and there needs to be a dependency on invoice.